utils/config.env

```diff
@@ -1,6 +1,28 @@
 #####################################################
-# Values in this file will be used to replace variables in script files
-# and Docker Compose files throughout the repo
+# The following values are provided by Confluent's
+# release automation process.  The values in these
+# names will be adopted in scripts and expressions
+# over time to construct other
+# values required by this repository
+#####################################################
+CONFLUENT=5.5.0-SNAPSHOT
+CONFLUENT_DOCKER_TAG=5.5.x-latest
+CONFLUENT_SHORT=5.5
+CONFLUENT_PREVIOUS=5.4.2
```

Comment:
1. Why is PREVIOUS necessary?
2. Is this by branch or date? e.g. Let's say 5.3.N is released after 5.4.N-1, then what is this value?

Reply:
I'm not certain all of them are necessary, but what was discussed in the companion document was that we would accept all available values from the tools automation here and use what's required as we go.

Reply:
@andrewegel can you answer @ybyzek 's question about PREVIOUS?

Reply:
As we are pulling these values into a public-facing repo, we should have clarity on what they represent.  @andrewegel could you elaborate on the question regarding PREVIOUS?  Thanks!

Reply:
`CONFLUENT_PREVIOUS` is just the previous version - take it or leave it, its your choice.

Reply:
As in this is the table:

`5.4.1` -> `5.4.0`
`5.4.0` -> `` (empty string)

Reply:
Do you know what specifically defines the value?

Reply:
https://github.com/confluentinc/release-tools/blob/master/confluent/release/versions.py#L692

Reply:
@andrewegel thanks for the link.  Based on

```
    Return the previous patch version for the given version without any suffix, or None if this is the first
```

IIUC since there is no 5.5.0 yet, then this should be set to `None`

Reply:
I've updated the value here to empty string per @andrewegel comment above